src/components/JuiceVideo/JuiceVideoThumbnailOrImage.tsx

```diff
@@ -45,41 +42,43 @@ export function JuiceVideoThumbnailOrImage({
         >
           <LoadingOutlined className="text-primary" />
         </div>
-      ) : null}
-      <div
-        onClick={
-          showPreviewOnClick
-            ? stopPropagation(() => setPreviewVisible(true))
-            : undefined
-        }
-        className="h-full w-full"
-      >
-        {isVideo ? (
-          <JuiceVideoThumbnail
-            src={props.src}
-            videoClassName={props.className}
-            onLoaded={() => setLoading(false)}
-            playIconPosition={playIconPosition}
-          />
-        ) : (
-          <img
-            className={`${
-              props.className ?? ''
-            } top-0 h-full w-full object-cover`}
-            src={props.src}
-            onClick={props.onClick}
-            crossOrigin="anonymous"
-            onLoad={() => setLoading(false)}
-            alt="Enlarged media preview"
-          />
-        )}
-      </div>
+      )}
+      {isVideo ? (
```

Comment:
@johnnyd-eth I squashed everything down into a single div, rather than the nested divs with h-full/w-full. This seemed to solve most of the issues. No need for `containerClass` anymore either